pkg/chain/gen/celo/abi/BondedECDSAKeep.go

```diff
@@ -0,0 +1,2154 @@
+// Code generated - DO NOT EDIT.
```

Comment:
I am confused why do we add ABI for `BondedECDSAKeep.go` but we do not for `BondedECDSAKeepFactory.go` and everything works. 🤔

Reply:
It's just to make `gosec` pass. We do the same for `pkg/chain/gen/ethereum/abi/BondedECDSAKeep.go`.